phase1/gce/configure-vm-kubeadm.sh

```diff
@@ -4,6 +4,10 @@ TOKEN=$(get_metadata "k8s-kubeadm-token")
 KUBEADM_VERSION=$(get_metadata "k8s-kubeadm-version")
 KUBERNETES_VERSION=$(get_metadata "k8s-kubernetes-version")
 KUBELET_VERSION=$(get_metadata "k8s-kubelet-version")
+KUBEADM_DIR=/etc/kubeadm
+KUBEADM_INIT_PARAM_FILE=$KUBEADM_DIR/kubeadm_init_params.txt
+KUBELET_VERSION=$(get_metadata "k8s-kubelet-version")
```

Comment:
rebase conflict?

Reply:
Bad merge on my end. Let me remove the extra kubelet version.